README.md

```diff
@@ -15,32 +15,21 @@ It will look at the pages API for your site, find all pages that have expired, a
 
 ### `alphagov` users
 
-If you are part of the `alphagov` GitHub organisation you can enable the notifier by raising a PR to add your published documentation to the [`.travis.yml` file][travis]:
+If you are part of the `alphagov` GitHub organisation you can enable the notifier by raising a PR to add your published documentation to the [`Rakefile`][Rakefile]:
 
 ```
-matrix:
-  - SITE_PAGE_API_URL=https://www.docs.verify.service.gov.uk/api/pages.json
-  - SITE_PAGE_API_URL=https://gds-way.cloudapps.digital/api/pages.json
-  - SITE_PAGE_API_URL=https://<YOUR_PUBLISHED_DOCS>/api/pages.json
+pages_urls = [
+  "https://gds-way.cloudapps.digital/api/pages.json",
+  "https://docs.publishing.service.gov.uk/api/pages.json",
+  "your-docs-site.cloudapps.digital"
+]
 ```
 
-[travis]: https://github.com/alphagov/tech-docs-monitor/blob/master/.travis.yml
-
-### Configure Travis CI
-
-If you are not part of the `alphagov` GitHub organisation, you can still configure Travis CI to automatically deploy the notifier:
-
-1. Fork the `tech-docs-monitor` repository
-1. Get an [incoming Slack webhook][webhook] for the notifier
-1. Run [`travis encrypt`][encrypt] to add the encrypted webhook to your `.travis.yml` file
-1. Add your published documentation page API URL to `env.matrix` in your `.travis.yml` file
-
-[encrypt]: https://docs.travis-ci.com/user/encryption-keys/#usage
-[webhook]: https://api.slack.com/incoming-webhooks
+[Rakefile]: https://github.com/alphagov/tech-docs-monitor/blob/master/Rakefile
 
 ### General configuration
 
-If you want to use something other than Travis CI to deploy the notifier, you must make sure all its environment variables are defined:
+The following environment variables are necessary:
 
 * `SITE_PAGE_API_URL`: The full URL to your site's `/api/pages.json` file.
```

Comment:
I thought we [removed this one](https://github.com/alphagov/tech-docs-monitor/pull/19/commits/08d081f3b843bcd8be4656d99c60dce81fbbbed4)?